packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenuPage.tsx

```diff
@@ -72,7 +72,7 @@ export class ContextualMenuPage extends React.Component<IComponentDemoPageProps,
               <ContextualMenuSectionExample />
             </ExampleCard>
             <ExampleCard
-              title='ContextualMenu with checkable menu items and toggable split button'
+              title='ContextualMenu with checkable menu items and toggleable split button'
               code={ ContextualMenuCheckmarksExampleCode }
```

Comment:
I know this isn't a word anyway, but I think this is the correct spelling.

Reply:
According to some random people on the internet: https://english.stackexchange.com/questions/232774/is-it-togglable-or-toggleable

"The English language's main strength is its adaptability, so one day it will most likely be officially accepted as a word, but for now, it has no defined spelling."

Who knows, it might be correct in the near future!